zproject/urls.py

```diff
@@ -354,6 +355,8 @@
     # typing -> zerver.views.typing
     # POST sends a typing notification event to recipients
     rest_path("typing", POST=send_notification_backend),
+    # read_by -> zerver.views.read_indicators
+    rest_path("read_by/<int:message_id>", GET=read_by_users),
```

Comment:
This should be something like `GET /message/{message_id}/readers`, to follow `REST` conventions.

Reply:
Ok, changed. Thanks, didn't know.